src/main/java/de/hysky/skyblocker/skyblock/dungeon/CroesusProfit.java

```diff
@@ -82,7 +79,7 @@ private double valueChest(@NotNull ItemStack chest) {
             } else if (lineString.isEmpty()) {
                 processingContents = false;
             } else if (lineString.contains("Coins") && !processingContents) {
-                chestPrice = Integer.parseInt(lineString.replaceAll(",", "").replaceAll("\\D", ""));
+                chestPrice = Integer.parseInt(lineString.replace(",", "").replaceAll("\\D", ""));
```

Comment:
Is this intentional?

Reply:
Yes, iirc this was because `replaceAll` compiles the given string into a regex to find occurrences of it whereas `replace` simply does string comparison